LoopKit/TherapySettings.swift

```diff
@@ -8,7 +8,7 @@
 
 import HealthKit
 
-public struct TherapySettings: Equatable {
+public struct TherapySettings: Equatable, Codable {
```

Comment:
These changes are to enable `TherapySettings` to be a subcomponent of a prescription